mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/DevOPS-Test-Module-Selenium-Grid.asciidoc

```diff
@@ -1,3 +1,19 @@
+:toc: macro
+
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+toc::[]
+:idprefix:
+:idseparator: -
+:reproducible:
+:source-highlighter: rouge
+:listing-caption: Listing
 = Selenium Grid
 
 == What is Selenium Grid
```

Comment:
The last screenshot in this section can show selenium grid in the newest version (version 4)